docs/plugins/mbsubmit.rst

```diff
@@ -1,23 +1,36 @@
 MusicBrainz Submit Plugin
 =========================
 
-The ``mbsubmit`` plugin provides an extra prompt choice during an import
-session and a ``mbsubmit`` command that prints the tracks of the current
-album in a format that is parseable by MusicBrainz's `track parser`_.
+The ``mbsubmit`` plugin provides extra prompt choices when an import session
+fails to find a good enough match for a release. Additionally, it provides an
+``mbsubmit`` command that prints the tracks of the current album in a format
+that is parseable by MusicBrainz's `track parser`_. The prompt choices are:
+
+- Print the tracks to stdout in a format suitable for MusicBrainz's `track
+  parser`_.
+
+- Print the tracks into a ``beets-unmatched`` file in the same format as in the
+  above.
+
+- Open the program `Picard`_ with the unmatched folder as an input, allowing
```

Comment:
I am assuming this requires Picard to be installed. Please clarify that and link to the install docs.

Reply:
> I am assuming this requires Picard to be installed. Please clarify that and link to the install docs.

Thanks, good idea. Did something.